Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5855,6 +5856,78 @@ def com_google_fonts_check_metadata_can_render_samples(ttFont, family_metadata):
        yield PASS, "OK."
 
 
+def count_islands(imagefont, character):
+    from PIL import Image, ImageDraw
+    width, height = imagefont.getsize(character)
+    img = Image.new("RGBA", (width, height))
+    draw = ImageDraw.Draw(img)
+    draw.text((-2, 0), str(character), font=imagefont, fill="#000000")
+
+    visited = [False] * (width * height)
+    threshold = 150
+
+    def mark_visted(x, y):
+        if (x < 0) or (x >= width) or (y < 0) or (y >= height) \
+           or img.getpixel((x, y))[3] <= threshold \
+           or visited[y*width + x]:
+            return
```

Comment:
Also, I was not sure why I was only seeing alpha values changing, when I wrote this code. But now, reviewing my own code to start a conversation with @m4rc1e I think it may be the case that I am rendering black text on a black background, perhaps? I am not sure what's the default background color in a Pillow canvas...